src/Compilers/CSharp/Portable/Binder/Binder.cs

```diff
@@ -158,6 +158,14 @@ internal virtual ImmutableArray<LocalSymbol> GetDeclaredLocalsForScope(CSharpSyn
         }
 
         /// <summary>
+        /// Get local functions declared immediately in scope represented by the node.
+        /// </summary>
+        internal virtual ImmutableArray<LocalFunctionSymbol> GetDeclaredLocalFunctionsForScope(CSharpSyntaxNode node)
```

Comment:
It could be leftovers from declaration expressions or from single meaning enforcement. 
I am for removing this parameter from both methods. It should make the code simpler.